pkg/cloud/aks/mocks/azurestorage.go

```diff
@@ -0,0 +1,196 @@
+// Code generated by pegomock. DO NOT EDIT.
+// Source: github.com/jenkins-x/jx/v2/pkg/cloud/aks (interfaces: AzureStorage)
+
+package aks_test
+
+import (
```

Comment:
So, this is just one of the side-effects of using pegomock, you need to fix the goimport issue (sadly in a file that should not be edited by hand), for the codegen tests to pass. I have an [issue](https://github.com/jenkins-x/jx/issues/7307) to replace pegomock with gomock.